<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
<hTML xmlns="http://www.w3.org/TR/REC-html40" xmlns:v =
"urn:schemas-microsoft-com:vml" xmlns:o =
"urn:schemas-microsoft-com:office:office" xmlns:w =
"urn:schemas-microsoft-com:office:word"><head><title>EBI Research - Functional Genomics - Introduction To Biology</title>
<meta http-equiv=Content-Type content="text/html; charset=iso-8859-1">
<meta content=Word.Document name=ProgId>
<meta content="MSHTML 5.50.4912.300" name=GENERATOR>
<meta content="Microsoft Word 9" name=Originator><!--[if !mso]>
<style>v\:* {
	BEHAVIOR: url(#default#VML)
}
o\:* {
	BEHAVIOR: url(#default#VML)
}
w\:* {
	BEHAVIOR: url(#default#VML)
}
.shape {
	BEHAVIOR: url(#default#VML)
}
</style>
<![endif]-->
<style>@page Section1 {size: 595.3pt 841.9pt; margin: 72.0pt 90.0pt 72.0pt 90.0pt; mso-header-margin: 35.4pt; mso-footer-margin: 35.4pt; mso-paper-source: 0; }
P.MsoNormal {
	FONT-SIZE: 12pt; margin: 0cm 0cm 0pt; FONT-FAMILY: "Times New Roman"; mso-style-parent: ""; mso-pagination: widow-orphan; mso-fareast-font-family: "Times New Roman"
}
LI.MsoNormal {
	FONT-SIZE: 12pt; margin: 0cm 0cm 0pt; FONT-FAMILY: "Times New Roman"; mso-style-parent: ""; mso-pagination: widow-orphan; mso-fareast-font-family: "Times New Roman"
}
DIV.MsoNormal {
	FONT-SIZE: 12pt; margin: 0cm 0cm 0pt; FONT-FAMILY: "Times New Roman"; mso-style-parent: ""; mso-pagination: widow-orphan; mso-fareast-font-family: "Times New Roman"
}
H1 {
	FONT-WEIGHT: bold; FONT-SIZE: 24pt; margin-left: 0cm; margin-right: 0cm; FONT-FAMILY: "Times New Roman"; mso-pagination: widow-orphan; mso-margin-top-alt: auto; mso-margin-bottom-alt: auto; mso-outline-level: 1; mso-font-kerning: 18.0pt
}
H2 {
	FONT-WEIGHT: bold; FONT-SIZE: 18pt; margin-left: 0cm; margin-right: 0cm; FONT-FAMILY: "Times New Roman"; mso-pagination: widow-orphan; mso-margin-top-alt: auto; mso-margin-bottom-alt: auto; mso-outline-level: 2
}
H3 {
	FONT-WEIGHT: bold; FONT-SIZE: 13.5pt; margin-left: 0cm; margin-right: 0cm; FONT-FAMILY: "Times New Roman"; mso-pagination: widow-orphan; mso-margin-top-alt: auto; mso-margin-bottom-alt: auto; mso-outline-level: 3
}
P.MsoBodyText {
	FONT-SIZE: 12pt; margin: 0cm 0cm 6pt; FONT-FAMILY: "Times New Roman"; text-align: justify; mso-pagination: widow-orphan; mso-fareast-font-family: "Times New Roman"; mso-ansi-language: EN-US
}
LI.MsoBodyText {
	FONT-SIZE: 12pt; margin: 0cm 0cm 6pt; FONT-FAMILY: "Times New Roman"; text-align: justify; mso-pagination: widow-orphan; mso-fareast-font-family: "Times New Roman"; mso-ansi-language: EN-US
}
DIV.MsoBodyText {
	FONT-SIZE: 12pt; margin: 0cm 0cm 6pt; FONT-FAMILY: "Times New Roman"; text-align: justify; mso-pagination: widow-orphan; mso-fareast-font-family: "Times New Roman"; mso-ansi-language: EN-US
}
P.MsoBodyTextIndent {
	FONT-SIZE: 12pt; margin: 0cm 0cm 0pt 36pt; FONT-FAMILY: "Times New Roman"; mso-pagination: widow-orphan; mso-fareast-font-family: "Times New Roman"
}
LI.MsoBodyTextIndent {
	FONT-SIZE: 12pt; margin: 0cm 0cm 0pt 36pt; FONT-FAMILY: "Times New Roman"; mso-pagination: widow-orphan; mso-fareast-font-family: "Times New Roman"
}
DIV.MsoBodyTextIndent {
	FONT-SIZE: 12pt; margin: 0cm 0cm 0pt 36pt; FONT-FAMILY: "Times New Roman"; mso-pagination: widow-orphan; mso-fareast-font-family: "Times New Roman"
}
P.MsoBodyText2 {
	FONT-SIZE: 11pt; margin: 0cm 0cm 6pt; FONT-FAMILY: "Times New Roman"; text-align: justify; mso-pagination: widow-orphan; mso-fareast-font-family: "Times New Roman"; mso-bidi-font-size: 12.0pt
}
LI.MsoBodyText2 {
	FONT-SIZE: 11pt; margin: 0cm 0cm 6pt; FONT-FAMILY: "Times New Roman"; text-align: justify; mso-pagination: widow-orphan; mso-fareast-font-family: "Times New Roman"; mso-bidi-font-size: 12.0pt
}
DIV.MsoBodyText2 {
	FONT-SIZE: 11pt; margin: 0cm 0cm 6pt; FONT-FAMILY: "Times New Roman"; text-align: justify; mso-pagination: widow-orphan; mso-fareast-font-family: "Times New Roman"; mso-bidi-font-size: 12.0pt
}
P.MsoBodyText3 {
	FONT-SIZE: 11pt; margin: 0cm 0cm 0pt; FONT-FAMILY: "Times New Roman"; mso-pagination: widow-orphan; mso-fareast-font-family: "Times New Roman"; mso-bidi-font-size: 12.0pt
}
LI.MsoBodyText3 {
	FONT-SIZE: 11pt; margin: 0cm 0cm 0pt; FONT-FAMILY: "Times New Roman"; mso-pagination: widow-orphan; mso-fareast-font-family: "Times New Roman"; mso-bidi-font-size: 12.0pt
}
DIV.MsoBodyText3 {
	FONT-SIZE: 11pt; margin: 0cm 0cm 0pt; FONT-FAMILY: "Times New Roman"; mso-pagination: widow-orphan; mso-fareast-font-family: "Times New Roman"; mso-bidi-font-size: 12.0pt
}
A:link {
	COLOR: blue; TEXT-DECORATION: underline; text-underline: single
}
SPAN.MsoHyperlink {
	COLOR: blue; TEXT-DECORATION: underline; text-underline: single
}
A:visited {
	COLOR: blue; TEXT-DECORATION: underline; text-underline: single
}
SPAN.MsoHyperlinkFollowed {
	COLOR: blue; TEXT-DECORATION: underline; text-underline: single
}
P {
	FONT-SIZE: 12pt; margin-left: 0cm; margin-right: 0cm; FONT-FAMILY: "Times New Roman"; mso-pagination: widow-orphan; mso-fareast-font-family: "Times New Roman"; mso-margin-top-alt: auto; mso-margin-bottom-alt: auto
}
ADDRESS {
	FONT-SIZE: 12pt; margin: 0cm 0cm 0pt; FONT-STYLE: italic; FONT-FAMILY: "Times New Roman"; mso-pagination: widow-orphan
}
PRE {
	FONT-SIZE: 10pt; margin: 0cm 0cm 0pt; FONT-FAMILY: "Courier New"; mso-pagination: widow-orphan; mso-fareast-font-family: "Courier New"; tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt
}
TT {
	mso-fareast-font-family: "Courier New"; mso-ascii-font-family: "Courier New"; mso-hansi-font-family: "Courier New"; mso-bidi-font-family: "Courier New"
}
DIV.Section1 {
	page: Section1
}
OL {
	margin-bottom: 0cm
}
UL {
	margin-bottom: 0cm
}
</style>
<!--[if gte mso 9]><xml>
 <u1:DocumentProperties>
  <u1:Author>brazma</u1:Author>
  <u1:Template>Normal</u1:Template>
  <u1:LastAuthor>brazma</u1:LastAuthor>
  <u1:Revision>11</u1:Revision>
  <u1:TotalTime>3188</u1:TotalTime>
  <u1:Created>2001-10-01T17:25:00Z</u1:Created>
  <u1:LastSaved>2001-10-01T17:50:00Z</u1:LastSaved>
  <u1:Pages>18</u1:Pages>
  <u1:Words>7187</u1:Words>
  <u1:Characters>40971</u1:Characters>
  <u1:Company>European Bioinformatics Institute EMBL-EBI</u1:Company>
  <u1:Lines>341</u1:Lines>
  <u1:Paragraphs>81</u1:Paragraphs>
  <u1:CharactersWithSpaces>50315</u1:CharactersWithSpaces>
  <u1:Version>9.3821</u1:Version>
 </u1:DocumentProperties>
</xml><![endif]--><!--[if gte mso 9]><xml>
 <u2:shapedefaults u3:ext="edit" spidmax="1072"/>
</xml><![endif]--><!--[if gte mso 9]><xml>
 <u4:shapelayout u5:ext="edit">
  <u4:idmap u5:ext="edit" data="1"/>
 </u4:shapelayout>
</xml><![endif]--></head>
<body lang=EN-GB style="tab-interval: 36.0pt" vLink=blue link=blue>
<div class=Section1>
<p class=MsoNormal><img id=_x0000_i1025 height=77
src="biology_intro_files/ebibanner.gif" width=627></p>
<h1><a name="top"></a>A quick introduction to elements of biology - cells,
molecules, genes, functional genomics, microarrays</h1>
<p class=MsoNormal> </p>
<aDDRESS>Alvis Brazma, Helen Parkinson, Thomas Schlitt, Mohammadreza
Shojatalab</ADDRESS>
<p class=MsoNormal> </p>
<p class=MsoNormal style="text-align: right" align="right">Draft, October 2001</p>
<p class=MsoNormal>This is a brief introduction to molecular biology with
emphasis on genomics and bioinformatics. It is intended for scientists,
engineers, computer programmers, or anybody with background or strong interest
in science, but without background in biology, and first of all for those
joining the <a href="http://www.ebi.ac.uk/">EBI</a> . On one hand we have tried
to distil the content down to the absolute minimum needed to make some sense of
bioinformatics, while on the other to leave in enough to show why it is
interesting. <br />&nbsp; </p>
<p class=MsoNormal><I>Disclaimer: </I>Most biologists agree that there are very
few strict rules in biology and that most rules have exceptions. Therefore, what
is written here may not always be strictly true.</p>
<p class=MsoNormal style="margin-left: 36pt"> </p>
<h3>Content</h3>
<p class=MsoNormal><span style="FONT-SIZE: 13.5pt"><a
href="#Organisms">1.
Organisms and cells</a> </span><br /><span style="FONT-SIZE: 13.5pt"><a
href="#Molecules">2.
Molecules of life</a> </span></p>
<p class=MsoNormal><span style="FONT-SIZE: 13.5pt"><a
href="#Genomes">3. Genes
and genomes</a> </span></p>
<p class=MsoNormal><span style="FONT-SIZE: 13.5pt"><a
href="#Functional">4.
Functional genomics</a> </span></p>
<p class=MsoNormal><span style="FONT-SIZE: 13.5pt"><a
href="#Microarrays">5.
Microarrays and gene expression databases</a> </span></p>
<h2><br />1.&nbsp;<a name=Organisms></a> Organisms and cells</h2>
<p class=MsoNormal>All organisms consist of small <I>cells,</I> typically too
small to be seen by a naked eye, but big enough for an optical <a
href="biology_intro_files/microscopy.htm">microscope</a>
. Each cell is a complex system consisting of many different building blocks
enclosed in <I>membrane</I> bag. There are unicellular (consisting only of one
cell) and multicellular organisms. Bacteria and baker&#8217;s yeast are examples of
unicellular organisms - any one cell is able to survive and multiply
independently in appropriate environment.</p>
<p class=MsoNormal>There are estimated about 6x10<sup>13</sup> cells in a human
body, of about 320 different types. For instance there are several types of skin
cells, muscle cells, brain cells (neurons), among many others. The number of
cell types is not well-defined, it depends on the similarity threshold (what
level of detail we would like to use to distinguish between the cell types,
e.g., it is unlikely that we would be able to find two identical cells in an
organism if we count the number of their molecules). The cell sizes may vary
depending on the cell type and circumstances. For instance, a human red blood
cell is about 5 microns (0.005 mm) in diameter, while some neurons are about 1 m
long (from spinal cord to leg). Typically the diameter of animal and plant cells
are between 10 and 100 microns. </p>
  <p>There are two types of organisms - <I>eukaryotes</I> and <I>prokaryotes,
</I>and<I> </I>two types of cells respectively. Bacteria belong to the
prokaryotes. However, most organisms which we can see, such as trees, grass,
flowers, weeds, worms, flies, mice, cats, dogs, humans, mushrooms and yeast are
eukaryotes. The distinction between eukaryotes and prokaryotes is rather
important, because many of the cellular building blocks and life processes are
quite different in these two organism types. This is believed to be the result
of different evolutionary paths. Evolution is an important concept in biology,
there is a proverb saying that things only make sense in biology in the context
of evolution. Most scientists believe that life first emerged on Earth around
3.8 billion years ago. The oldest fossilised bones that have been found
resembling bones from anatomically modern humans are about 100,000 &#8211; 200,000
years old. Nobody really knows how life emerged on Earth, but there is lots of
scientific evidence regarding how it may have evolved. </p>
<p><I>Viruses</I> are not quite living organisms, but when inside a living host
cell they show some features of a living organism. Viruses are too small to be
seen in an optical microscope, but are big enough to reveal their <a
href="biology_intro_files/virus.htm">structure</a> in an
electron microscope (the characteristic size of the virus is about 0.05-0.1
micron, while the wavelength of green light is about 0.5 micron). </p>
<p style="margin-bottom: 12pt"><a
href="http://www.ncbi.nlm.nih.gov/books/bv.fcgi?call=bv.View..ShowSection&amp;searchterm=procaryotic&amp;rid=cell.section.d1e389#d1e402">Prokaryotic
cells</a> are smaller than eukaryotic cells (a typical size of a prokaryotic
cell is about 1 micron in diameter) and have simpler structure (e.g., they do
not have any inner cellular membranes that are always present in Eukaryotes, see
below). Prokaryotes are single cellular organisms, but note that being a single
cell does not mean that an organism is a prokaryote. Being smaller than
eukaryotes does not mean that prokaryotes are any less important &#8211; for instance
it is quite likely that the number of bacteria living in the mouth and digestive
tract of a human&nbsp; are larger than the number of eukaryotic cells in the
same individual and many of these bacteria are necessary for a human being to
live a normal life (these numbers are rather difficult to estimate, rather a
hypothesis). Prokaryotes are sometimes also known as microbes. <br />&nbsp;
<br />&nbsp; </p>
<p style="text-align: center" align="center"><img id=_x0000_i1070 height=311
alt="eukaryotic cell" src="biology_intro_files/cell.jpe" width=432
border="0"></p>
<p class=MsoNormal>A model of a eukaryotic cell (picture taken from <a
href="http://gened.emc.maricopa.edu/bio/bio181/BIOBK/BioBookTOC.html">On-Line
Biology Book</a> ) </p>
<p>A <a
href="http://www.ncbi.nlm.nih.gov/books/bv.fcgi?call=bv.View..ShowSection&amp;searchterm=eucaryotic&amp;rid=cell.section.d1e389#d1e657">eukaryotic
cell</a> has a nucleus, which is separated from the rest of the cell by a
membrane. The nucleus contains chromosomes, which are the carrier of the genetic
material (Section 3). There are internal membrane enclosed compartments within
eukaryotic cells, called <I>organelles</I>, e.g., centrioles, lysosomes, golgi
complexes, mitochondria among others (see picture above), which are specialised
for particular biological processes. The <I>mitochondria</I> are found in all
eukaryotes and are specialised for energy production (respiration). Chloroplasts
are organelles found in plant cells which produce sugar using light. Light is
the ultimate source of energy for almost all life on Earth. The area of the cell
outside the nucleus and the organelles is called the cytoplasm. Membranes are
complex structures and they are an effective barrier to the environment, and
regulate the flow of food, energy and information in and out of the cell. There
is a theory that mitochondria are prokaryotes living within eukaryotic
cells.</p>
<p>An essential feature of most (prokaryote and eukaryote) living cells is their
ability to grow in an appropriate environment and to undergo cell division. The
growth of a single cell and its subsequent division is called the <I>cell
cycle</I>. However, not all cells continually grow and divide, for example
neurons only undergo an initial growth phase. Prokaryotes, particularly
bacteria, are extremely successful at multiplying - it is likely that natural
selection has favoured single celled organisms able to grow and divide quickly.
Multicellular organisms typically begin life as a single cell, usually as a
result of fusion of a male and a female sex cell (gametes). The single cell has
to grow, divide and <I>differentiate</I> into different cell types to produce
tissues and in higher eukarotyes, organs. Cell division and differentiation need
to be controlled. Cancerous cells grow without control and can go on to form
tumours. Development of single cells into complex organisms is in itself an area
of study called developmental biology. This year&#8217;s<span
style="mso-spacerun: yes">&nbsp; </span><a
href="http://www.nobel.se/medicine/laureates/2001/press.html">Nobel prize</a>
for Physiology or Medicine has been awarded to scientists for the discoveries of
key regulators of the cell cycle. <br /><br />Cells consist of molecules. <br />&nbsp;
</p>
<h2><a name=Molecules></a>2. Molecules of life</h2>
<p>There are four basic types of molecules involved in life: (1) small
molecules, (2) proteins, (3) DNA and (4) RNA. Proteins, DNA and RNA are known
collectively as <I>biological macromolecules</I>. </p>
<h3>2.1. Small molecules</h3>
<p class=MsoNormal style="margin-bottom: 12pt">These can be the building blocks
of the macromolecules or they can have independent roles, such as signal
transmission or being a source of energy or material for a cell. Some important
examples besides water are <I>sugars, fatty acids, amino acids</I> and
<I>nucleotides</I>. For instance, biological membranes are constructed from
fatty acids, into which macromolecules are embedded. There are 20 different
amino acid molecules, which are the building blocks for <I>proteins </I>(to be
more precise, there are 19 amino acids and one which has a slightly different
structure and therefore is called imino acid). <br />&nbsp;</p>
<p style="text-align: center" align="center"><img id=_x0000_i1078 height=250
src="biology_intro_files/aas_1.gif" width=411 border="0"></p>
<p class=MsoBodyText3>These are three examples of amino acid moleclues, there
are 17 more. They differ by R side chains which determine their properties and
the order of these different amino acids within the protein determines the three
dimensional structure of the protein. There is a convention that each amino-acid
is denoted by a letter in Latin alphabet, for instance arginine&nbsp; is denoted
by R, histidine by H, lysine by L and there are <a
href="http://rsc.anu.edu.au/~drsnag/Amino-acids.html">20 such letters</a> . </p>
<h3>2.2 Proteins</h3>
<p class=MsoNormal>Proteins are the main building blocks and functional
molecules of the cell, taking up almost 20% of a eukaryotic cell&#8217;s weight, the
largest contribution after water (70%). Among others, there are </p>
<ul type=disc>
  <li class=MsoNormal
  style="mso-margin-top-alt: auto; mso-margin-bottom-alt: auto; tab-stops: list 36.0pt; mso-list: l2 level1 lfo3">Structural
  proteins, which can be thought of as the organism's basic building blocks. An
  example is collagen, which is the major structural protein of connective
  tissue and bone.
  <li class=MsoNormal
  style="mso-margin-top-alt: auto; mso-margin-bottom-alt: auto; tab-stops: list 36.0pt; mso-list: l2 level1 lfo3">Enzymes,
  which perform (catalyse) a multitude of biochemical reactions, such as
  altering, joining together or chopping up other molecules. Together these
  reactions and the pathways they make up is called <I>metabolism</I>. For
  example the first step in the glycolysis pathway, which is the conversion of
  glucose to glucose 6-phosphate, is catalysed by the enzyme hexokinase. Usually
  enzymes are very specific and catalyse only a single type of reaction, however
  the same enzyme can play role in more than one pathway.
  <li class=MsoNormal
  style="mso-margin-top-alt: auto; mso-margin-bottom-alt: auto; tab-stops: list 36.0pt; mso-list: l2 level1 lfo3">Transmembrane
  proteins are key in maintenance of the cellular environment, regulating cell
  volume, extraction and concentration of small molceules from the extracellular
  environment and generation of ionic gradients essential for muscle and nerve
  cell function. An example is the sodium/potassium pump.<o:p></o:p> </li></ul>
<p class=MsoNormal style="margin-bottom: 6pt; text-align: justify"><span
lang=EN-US style="mso-ansi-language: EN-US"><U1:p></U1:p><br />Proteins have
complex three dimensional (3D) structure (see figure below). Four levels of
protein structure are distinguishable:<U1:p></U1:p></span></p>
  <ol type=1>
    <li class=MsoNormal
  style="mso-margin-top-alt: auto; tab-stops: list 36.0pt; mso-list: l1 level1 lfo6">Proteins
  are chains of 20 different types of amino acids, which in principle can be
  joined together in any linear order, sometimes called <I>poly-peptide</I>
  chains. This sequence of amino-acids is known as the <I>primary structure</I>,
  and it can be represented as a string of 20 different symbols&nbsp; (i.e., a
  word over the common alphabet of 20 letters). Information about various
  protein sequences and the functional roles of the respective proteins, can be
  found in <a href="http://www.ebi.ac.uk/swissprot/">UniProtKB/Swiss-Prot</a> database.
  UniProtKB/Swiss-Prot is a joint project between the EBI and the<a
  href="http://www.isb-sib.ch/"> Swiss Institute of Bioinformatics (SIB)</a>.
  The length of the protein molecule can vary from few to many thousands of
  amino-acids. For example <a
  href="http://srs.ebi.ac.uk/srsbin/cgi-bin/wgetz?-id+2ke3W1HJ1Ru+-e+%5BSWALL:%27INS_HUMAN%27%5D">insulin</a>
  is a small protein and it consists of 51 amino acids, while <a
  href="http://srs.ebi.ac.uk/srsbin/cgi-bin/wgetz?-id+2ke3W1HJ1_b+-e+%5BSWALL:%27Q10466%27%5D">titin</a>
  has ~28,000 amino acids.
    <li class=MsoNormal
  style="mso-margin-top-alt: auto; mso-margin-bottom-alt: auto; tab-stops: list 36.0pt; mso-list: l1 level1 lfo6">Although
  the primary structure of a protein is linear, the molecule is not straight,
  and the sequence of the amino acids affects the folding. There are two common
  substructures often seen within folded chains - <I>alpha-helices</I> and
  <I>beta-strands.</I> They are typically joined by less regular structures,
  called&nbsp; <I>loops</I>. These three are called <I>secondary structure
  elements</I>.
    <li class=MsoNormal
  style="mso-margin-top-alt: auto; mso-margin-bottom-alt: auto; tab-stops: list 36.0pt; mso-list: l1 level1 lfo6">As
  the result of the folding, parts of a protein molecule chain come into contact
  with each other and various attractive or repulsive forces (hydrogen bonds,
  disulfide bridges, attractions between positive and negative charges, and
  hydrophobic and hydrophilic forces) between such parts cause the molecule to
  adopt a fixed relatively stable 3D structure. This is called <I>tertiary</I>
  structure. In many cases the 3D structure is quite compact.
    <li class=MsoNormal
  style="mso-margin-top-alt: auto; mso-margin-bottom-alt: auto; tab-stops: list 36.0pt; mso-list: l1 level1 lfo6">A
  protein may be formed from more than one chain of amino-acids, in which case
  it is said to have <I>quaternary structure</I>. For example haemoglobin, is
  made up of four chains each of which is capable of binding an iron molecule.
  </li>
  </ol>
<p>Proteins are much too small to be seen in an optical microscope - a
characteristic protein size varies from about 3 to 10 nanometers (nm), i.e., 3
to 10 times 10<sup>-9 </sup>m, and solving (i.e., discovering) their structure
is a difficult and expensive exercise (approximately &#8364;50,000 - &#8364;200,000 per
novel structure), which is done by a variety of methods including X-ray
crystallography, nuclar-magnetic resonance spectroscopy, and advanced electron
microscopy. <a href="http://www.ebi.ac.uk/pdbe/">PDbe</a> is a database of known
protein structures, which is housed and developed at the EBI. The images below
shows the structure of triosephosphate isomerase visualised by RasMol software
package, a 3D viewer for PDBe structures. </p>
<p style="margin-bottom: 12pt; margin-left: 36pt"><img id=_x0000_i1030
height=400 src="biology_intro_files/rasmol.gif" width=350 border="0"><span
style="mso-tab-count: 1">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span><span style="mso-spacerun: yes">&nbsp;</span></p>
<p style="margin-bottom: 12pt"><span style="FONT-SIZE: 11pt">In this image the
magenta coloured bits are alpha-helices, while yellow bits are
beta-strands.<U6:p></U6:p></span> </p>
<p style="margin-bottom: 12pt; margin-left: 36pt"><img id=_x0000_i1031
height=400 src="biology_intro_files/triphos.jpe" width=400 border="0"></p>
  <p class=MsoBodyText><span lang=EN-US style="FONT-SIZE: 11pt">An alternative
view in which the two monomer units are highlighted. The size of this protein in
a crystallised state is about 13 x 7 x 5 nm. The images above are only models of
these molecules, as the molecules are two small to have a &#8216;real&#8217; image. For
instance they cannot have any conventional colour, they are in constant motion,
and when we start zooming in into a finer structure, quantum effects, such as
Heisenberg uncertainty principle start playing
role.&nbsp;<U1:p></U1:p></span></p>
<p class=MsoBodyText><span lang=EN-US>There are roughly 15,000 protein
structures deposited in public databases, though many of them are very similar
to each other. Whether to consider two protein structures&nbsp; similar or
different depends on the similarity threshold (as with cell types). Structural
biologists think that currently there are about 1,500 different representative
protein structures known.&nbsp;</span></p>
<p class=MsoBodyText><span lang=EN-US>All four structural levels are essentially
determined by the <I>primary structure</I> (i.e., the amino-acid sequence) plus
the physico-chemical environment where the molecule is placed. Predicting
protein structure from the amino-acid sequence is one of the most important
problems of computational biology (another name for bioinformatics, though some
try to make a distinction between these two terms) and is far from being solved.
Characteristic, frequently reoccurring structural elements are called <I>protein
domains</I>. Sometimes it is possible to identify these domains in proteins of
unknown structure, if their sequence is similar to that of a known structural
domain. Structural domains are often associated with a particular protein
function. Protein similarity is also deemed to be the result of evolutionary
relationship. </span></p>
<p>What are the comparative sizes of proteins and cells? There is a proverb
saying that size does not matter. Still comparative sizes may matter,
particularly if we try to imagine the cellular processes described in the next
sections. A typical linear dimension (diameter) of a globular protein is about 5
x 10<sup> -9</sup> m, while of a eukaryotic cell about 5 x 10<sup> -5</sup> m.
This means the a cell is about a 10,000 times larger than a protein linearly.
Alternatively, if we estimate the average weight of a human cell as about
10<sup> -9</sup> g, and remember that proteins constitute about one fifth of
cell mass, then assuming the weight of an average protein to be about 10<sup>
-19</sup> g (say hemoglobin is 64,500 atomic units, each of which is 1.66 x
10<sup> -24</sup> g), we see that there are 0.2 x 10<sup> -9</sup> / 10<sup>
-19</sup> proteins per cell, which equals two billion (2 x 10<sup> 9</sup> ).
These of course are very rough estimates which would vary from cell to cell. If
we remember that there are about 6 x 10<sup> 13</sup> cells, we see that there
are 30,000 times more cells per human, than proteins per cell. This may be an
indication of the relative complexity of a human compared to a single cellular
organism (a similar estimate regarding the relative complexity of an elephant or
dinosaur and human may not be flattering for a human).&nbsp;</p>
<p>Although forces such as hydrogen bonds are weak individually, when two or
more biological macromolecules with complementary shapes come close to each
other, the sum of all such weak forces may cause the molecules interact rather
strongly, e.g., to make them stick together. In fact, such weak inter-molecular
forces and interactions play a fundamental role in life and are at the basis of
virtually all biological processes. For instance many proteins can stick
together to form large <I>protein complexes</I> such as yeast RNA polymerase II,
which reads and transcribes the genetic information (see Section 3.3), and which
has 10 subunits and for which the <a
href="biology_intro_files/rnapol.gif">structure</a> has been solved
recently. These weak interactions also underlie how microarrays work, which is
discussed in the last section.&nbsp;</p>
<h3>2.3. DNA</h3>
<p class=MsoNormal>DNA is the main information carrier molecule in a cell. DNA
may be <I>single</I> or <I>double stranded</I>. A single stranded DNA molecule,
also called a <I>polynucleotide</I>, is a chain of small molecules, called
<I>nucleotides</I> . There are four different nucleotides grouped into two
types, purines: <I>adenosine</I> and <I>guanine</I> and pyrimidines:
<I>cytosine</I> and <I>thymine</I>. They are usually referred to as <I>bases</I>
(in fact bases are the only distinguishing element between different
nucleotides, see figure below) and denoted by their initial letters,
<b>A</b>,<b>C</b> ,<b>G</b> and <b>T </b>(not to be confused with amino acids!).
</p>
<p><img id=_x0000_i1032 height=138 src="biology_intro_files/nucleotides.gif"
width=522 border="0">(picture taken from&nbsp; <a
href="http://gened.emc.maricopa.edu/bio/bio181/BIOBK/BioBookTOC.html">On-Line
Biology Book</a> ) </p>
<p>Different nucleotides can be linked together in any order to form a
polynucleotide, for instance, like this<br
style="mso-special-character: line-break"><![if !supportLineBreakNewLine]><br
style="mso-special-character: line-break"><![endif]></p><pre style="margin-left: 36pt">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;A-G-T-C-C-A-A-G-C-T-T</pre>
<p class=MsoNormal
style="margin-top: 5pt; tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt">&nbsp;</p>
<p class=MsoNormal
style="margin: 5pt 0cm 12pt; tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt">Polynucleotides
can be of any length and can have any sequence. The two ends of this molecule
are chemically different, i.e., the sequence has a directionality, like this</p><pre style="margin: 5pt 0cm 0pt 36pt"><span style="mso-spacerun: yes">&nbsp;&nbsp;&nbsp;&nbsp; </span>A-&gt;G-&gt;T-&gt;C-&gt;C-&gt;A-&gt;A-&gt;G-&gt;C-&gt;T-&gt;T-&gt;</pre>
<p class=MsoNormal
style="margin-top: 5pt; tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt">&nbsp;</p>
<p class=MsoNormal
style="margin: 5pt 0cm 12pt; tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt">The
end of the polynucleotide are marked either 5' and 3' (this has chemical reasons
in the numbering of the &#8211;OH groups of the sugar ring); by convention DNA is
usually written with 5' left and 3' right, with the coding strand at top. Two
such strands are termed <I>complementary</I> , if one can be obtained from the
other by mutually exchanging A with T and C with G, and changing the direction
of the molecule to the opposite. For
instance,<U1:p></U1:p><o:p></o:p></p><U6:p></U6:p><pre style="margin: 5pt 0cm 12pt 36pt"><span style="mso-spacerun: yes">&nbsp;&nbsp;&nbsp;&nbsp; </span>&lt;-T&lt;-C&lt;-A&lt;-G&lt;-G&lt;-T&lt;-T&lt;-C&lt;-G&lt;-A&lt;-A<U1:p></U1:p><o:p></o:p></pre><U6:p></U6:p>
<p class=MsoNormal
style="margin-bottom: 6pt; text-align: justify; tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt"><span
lang=EN-US style="mso-ansi-language: EN-US">is complementary to the
polynucleotide given above. <br />Specific pairs of nucleotides can form weak
bonds between them. A binds to T, C binds to G (to be more precise, two hydrogen
bonds can be formed between each A-T pair, and three hydrogen bonds between each
C-G pair). Although such interactions are individually weak, when two longer
complementary polynucleotide chains meet, they tend to stick together, like
this<U1:p></U1:p><o:p></o:p></span></p>
<p class=MsoNormal
style="margin-bottom: 6pt; text-align: justify; tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt"><![if !supportEmptyParas]><![endif]>&nbsp;<o:p></o:p></p><pre style="margin: 6pt 0cm 0pt 36pt"><span style="mso-spacerun: yes">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span><tt>5' C-G-A-T-T-G-C-A-A-C-G-A-T-G-C 3'</tt></pre><pre style="margin-left: 36pt"><tt><span style="mso-spacerun: yes">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>| | | | | | | | | | | | | | | </tt></pre>
<p class=MsoNormal
style="margin-left: 36pt; tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt"><tt><span
style="FONT-SIZE: 10pt; FONT-FAMILY: 'Courier New'"><span
style="mso-spacerun: yes">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>3'
G-C-T-A-A-C-G-T-T-G-C-T-A-C-G 5'</span></tt> </p>
<p class=MsoNormal
style="tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt"><br />Vertical
lines between two strands represent the forces between them (to be more accurate
we could draw triple lines between each C and G and double lines between A and
T) as shown below. The A-T and G-C pairs are called <I>base-pairs</I> (bp). The
length of a DNA molecule is usually measured in base-pairs or nucleotides (nt),
which in this context is the same thing.&nbsp; </p>
<p class=MsoNormal
style="margin-left: 36pt; tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt">
</p>
<p class=MsoNormal
style="tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt"><br />&nbsp;
</p>
<p class=MsoNormal
style="margin-left: 36pt; text-align: center; tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt"
align="center"><img id=_x0000_i1033 height=300
src="biology_intro_files/dnach3.gif" width=444 border="0">(picture taken
from&nbsp; <a
href="http://gened.emc.maricopa.edu/bio/bio181/BIOBK/BioBookTOC.html">On-Line
Biology Book</a> )</p>
<p class=MsoNormal
style="tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt">
<br />Two complementary polynucleotide chains form a stable structure, which
resembles a helix and is known as a the DNA double helix. About 10 bp in this
structure takes a full turn, which is about 3.4 nm long.</p>
<p class=MsoNormal
style="margin-left: 36pt; tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt">
<br /><img id=_x0000_i1034 height=416 src="biology_intro_files/DNAhelix.gif"
width=177 border="0">(picture taken from&nbsp; <a
href="http://gened.emc.maricopa.edu/bio/bio181/BIOBK/BioBookTOC.html">On-Line
Biology Book</a> )</p>
<p class=MsoNormal
style="tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt">
<br />This structure was first figured out in 1953 in Cambridge by Watson and
Crick (with the help of others), and the birthplace of this structure is often
thought to be the <a
href="http://www.cambridge2000.com/cambridge2000/html/0009/P9042329.html">Eagle</a>
pub on Bene't street. Later they got the Nobel Prize for this discovery, for
more see the book by Watson &#8211; The Double Helix.</p>
<p class=MsoNormal
style="tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt"><![if !supportEmptyParas]><![endif]>&nbsp;<o:p></o:p></p>
<p class=MsoNormal
style="tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt"><U1:p></U1:p><img
id=_x0000_i1085 height=151 src="biology_intro_files/WatsonCrick.jpe" width=159
border="0"></p>
<p class=MsoNormal
style="tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt">Watson
and Crick at their DNA model molecule</p>
<p class=MsoNormal
style="tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt"><br />It
is remarkable that two complementary DNA polypeptides form a stable double helix
almost regardless of the sequence of the nucleotides. This makes the DNA
molecule a perfect medium for information storage. Note that as the strands are
complementary, each one of them fully determining the other, therefore for the
information purposes it is enough to give only one strand of the genome
molecules. Thus, for many information related purposes, the molecule used on the
example above, can be represented as CGATTCAACGATGC. The maximal amount of
information that can be encoded in such a molecule is therefore 2 bits times the
length of the sequence. Noting that the distance between nucleotide pairs in a
DNA is about 0.34 nm, we can calculate that the linear information storage
density in DNA is about 6x10<sup> 8</sup> bits/cm, which is approximately 75 GB
or 12.5 CD-Roms per cm.<o:p></o:p></p>
<p class=MsoNormal
style="tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt"><U1:p></U1:p><![if !supportEmptyParas]><![endif]>&nbsp;<o:p></o:p></p>
  <p class=MsoNormal
style="tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt">Complementarity
of two strands in the DNA is exploited for copying (multiplying) DNA molecules
in a process known as the <a
href="biology_intro_files/replication.html">DNA </a><I><a
href="biology_intro_files/replication.html">replication</a>
, </I>in which one double stranded DNA is replicated into two identical ones.
(The DNA double helix unwinds and forks during the process, and a new
complimentary strand is synthesised by specific molecular machinery on each
branch of the fork. After the process is finished there are two DNA molecules
identical to the original one.)<span style="mso-spacerun: yes">&nbsp;&nbsp;
</span>In a cell this happens during the cell division (see Section 1) and a
copy identical to the original goes to each of the new cells. <br />
    <br />Note that
mismatched components between polynucleotide strands are possible, if the total
sum of weak forces between the complementary nucleotides are strong enough. So
the molecules like</p>
<p class=MsoNormal
style="tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt"><br />&nbsp;
</p><pre style="margin-left: 36pt"><tt>C-G-A-T-T-G-C-C-A-C-G-A-T-G-C</tt></pre><pre style="margin-left: 36pt"><tt>| | | ~ | | | ~ | | | ~ | | |</tt></pre><pre style="margin-left: 36pt"><tt>G-C-T-T-A-C-G-T-T-G-C-A-A-C-G</tt></pre>
<p class=MsoNormal
style="tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt">
</p>
<p class=MsoNormal
style="margin-top: 5pt; tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt">are
chemically possible, though they may be rare in a living cell. More bonds, i.e.,
more complementary pairs, makes the molecule more stable. If there are not
enough bonds, the two stranded molecular structure may become weak and the
strands may come apart. The number of links needed to keep the double-helix
together depends on the temperature (so-called melting temperature) and other
environmental factors. DNA which is no longer in the helical form is said to be
denatured.</p>
<h3
style="tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt">2.4.
RNA</h3>
<p class=MsoNormal
style="margin-top: 5pt; tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt">RNA
like DNA is constructed from nucleotides. But instead of the pyrimidine thymine
(T), it has an alternative uracil (U), which is not found in DNA. Because of
this minor difference <a
href="biology_intro_files/RNA.html">RNA</a><span
style="mso-spacerun: yes">&nbsp; </span>do not form a double helix, instead
usually they are single stranded, but may have complex spatial structure due to
complementary links between the parts of the same strand (as for instance in <a
href="biology_intro_files/tRNA.htm">tRNA</a> , section 4.2).
RNA has various functions in a cell, some of them discussed in the next section,
e.g., mRNA and tRNA are functionally different types of RNA which are both
required for protein synthesis discussed in section 3.3.&nbsp;</p>
<p class=MsoNormal
style="margin-top: 5pt; tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt">RNA
can bind complementary to a single strand of a DNA molecule, even though T is
replaced by U, so molecules like this</p><pre style="margin: 5pt 0cm 0pt 36pt"><tt><span style="mso-spacerun: yes">&nbsp;&nbsp;&nbsp;&nbsp; </span>C-G-A-T-T-G-C-A-A-C-G-A-T-G-C<span style="mso-spacerun: yes">&nbsp;&nbsp; </span>DNA</tt></pre><pre style="margin-left: 36pt"><tt><span style="mso-spacerun: yes">&nbsp;&nbsp;&nbsp;&nbsp; </span>| | | | | | | | | | | | | | |</tt></pre><pre style="margin: 0cm 0cm 12pt 36pt"><tt> <span style="mso-spacerun: yes">&nbsp;&nbsp;&nbsp;&nbsp;</span>G-C-U-A-A-C-G-U-U-G-C-U-A-C-G<span style="mso-spacerun: yes">&nbsp;&nbsp; </span>RNA</tt></pre>
<p class=MsoNormal
style="tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt">are
possible and in fact play an important role in life processes and in
biotechnology.&nbsp;</p>
<p class=MsoNormal
style="tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt"><br />There
is a hypothesis that the first life on earth may have been RNA based. RNA can
encode genetic information, is replicable, forms complex 3D structures and can
also act as a catalyst for certain chemical reactions related to splicing (see
section 3.3).&nbsp; </p>
<h2
style="tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt">3.&nbsp;<a
name=Genomes></a> Genes and genomes</h2>
<h3
style="tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt">3.1
Chromosomes, genomes and sequencing</h3>
<p class=MsoNormal
style="margin-top: 5pt; tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt">In
a typical cell there are one or several long double stranded DNA molecules
organised as <I>chromosomes</I>. In eukaryotes chromosomes have a complex
structure where DNA is wound around structural proteins called histones. A human
has <a href="biology_intro_files/human_chromosomes.htm">23
pairs of chromosomes</a> , which are large enough to be seen in an optical
microscope. The total length of the DNA in one human cell, if we could stretch
it out, would be more than 1m. Mitochondria (section 1) contain DNA too, but the
amount is minuscule in comparison to <I>chromasomal </I>DNA. Chromasomal and
mitochondrial DNA forms the <I>genome</I> of the organism. All organisms have
genomes and they are believed to encode almost all the hereditary information of
the organism. In eukaryotes chromosoms are in the nucleus (apart from
mitochondrial genomes), contained by the nuclear membrane. All cells in an
organism contain identical genomes (with few rather special exceptions), as the
result of DNA replication at each cell division. </p>
<p
style="tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt">There
is a molecular machinery in cells, which keeps both DNA strands intact and
complementary (i.e., if one strand is damaged, it is repaired using the second
as a template). This is important as DNA damage (caused by environmental factors
like radiation) can result in breaks in one or both strands, or mispairing of
the bases, which would disrupt DNA replication among other things. If damaged
DNA is not repaired the result can be cell death or tumours. Changes in genomic
DNA are known as <I>mutations</I>. </p>
<p
style="tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt">The
total genome size differ quite considerably in different organisms, as given in
the table below. </p>
<table
style="width: 60%; mso-cellspacing: 1.5pt; mso-padding-alt: 0cm 0cm 0cm 0cm"
cellpadding="0" width="60%" border=1>
  
  <tr>
    <td
    style="padding-right: 1.5pt; padding-left: 1.5pt; padding-bottom: 1.5pt; padding-top: 1.5pt"
    vAlign=top>
      <p class=MsoNormal style="text-align: center" align="center">Organism</p></td>
    <td
    style="padding-right: 1.5pt; padding-left: 1.5pt; padding-bottom: 1.5pt; padding-top: 1.5pt"
    vAlign=top>
      <p class=MsoNormal style="text-align: center" align="center">Number
      or<br />chromosomes</p></td>
    <td
    style="padding-right: 1.5pt; padding-left: 1.5pt; padding-bottom: 1.5pt; padding-top: 1.5pt"
    vAlign=top>
      <p class=MsoNormal style="text-align: center" align="center">Genome size
      in<br />base pairs</p></td></tr>
  <tr>
    <td
    style="padding-right: 1.5pt; padding-left: 1.5pt; padding-bottom: 1.5pt; padding-top: 1.5pt"
    vAlign=top>
      <p class=MsoNormal style="text-align: center" align="center"><a
      href="http://www.sanger.ac.uk/Projects/Microbes/">Bacteria</a> </p></td>
    <td
    style="padding-right: 1.5pt; padding-left: 1.5pt; padding-bottom: 1.5pt; padding-top: 1.5pt"
    vAlign=top>
      <p class=MsoNormal style="text-align: center" align="center">1</p></td>
    <td
    style="padding-right: 1.5pt; padding-left: 1.5pt; padding-bottom: 1.5pt; padding-top: 1.5pt"
    vAlign=top>
      <p class=MsoNormal style="text-align: center" align="center">~400,000 -
      ~10,000,000</p></td></tr>
  <tr>
    <td
    style="padding-right: 1.5pt; padding-left: 1.5pt; padding-bottom: 1.5pt; padding-top: 1.5pt"
    vAlign=top>
      <p class=MsoNormal style="text-align: center" align="center"><a
      href="http://genome-www.stanford.edu/Saccharomyces/">Yeast</a> </p></td>
    <td
    style="padding-right: 1.5pt; padding-left: 1.5pt; padding-bottom: 1.5pt; padding-top: 1.5pt"
    vAlign=top>
      <p class=MsoNormal style="text-align: center" align="center">12</p></td>
    <td
    style="padding-right: 1.5pt; padding-left: 1.5pt; padding-bottom: 1.5pt; padding-top: 1.5pt"
    vAlign=top>
      <p class=MsoNormal style="text-align: center"
    align="center">14,000,000</p></td></tr>
  <tr>
    <td
    style="padding-right: 1.5pt; padding-left: 1.5pt; padding-bottom: 1.5pt; padding-top: 1.5pt"
    vAlign=top>
      <p class=MsoNormal style="text-align: center" align="center"><a
      href="http://www.sanger.ac.uk/Projects/C_elegans/">Worm</a> </p></td>
    <td
    style="padding-right: 1.5pt; padding-left: 1.5pt; padding-bottom: 1.5pt; padding-top: 1.5pt"
    vAlign=top>
      <p class=MsoNormal style="text-align: center" align="center">6</p></td>
    <td
    style="padding-right: 1.5pt; padding-left: 1.5pt; padding-bottom: 1.5pt; padding-top: 1.5pt"
    vAlign=top>
      <p class=MsoNormal style="text-align: center"
    align="center">100,000,000</p></td></tr>
  <tr>
    <td
    style="padding-right: 1.5pt; padding-left: 1.5pt; padding-bottom: 1.5pt; padding-top: 1.5pt"
    vAlign=top>
      <p class=MsoNormal style="text-align: center" align="center"><a
      href="http://edgp.ebi.ac.uk/">Fly</a> </p></td>
    <td
    style="padding-right: 1.5pt; padding-left: 1.5pt; padding-bottom: 1.5pt; padding-top: 1.5pt"
    vAlign=top>
      <p class=MsoNormal style="text-align: center" align="center">4</p></td>
    <td
    style="padding-right: 1.5pt; padding-left: 1.5pt; padding-bottom: 1.5pt; padding-top: 1.5pt"
    vAlign=top>
      <p class=MsoNormal style="text-align: center"
    align="center">300,000,000</p></td></tr>
  <tr>
    <td
    style="padding-right: 1.5pt; padding-left: 1.5pt; padding-bottom: 1.5pt; padding-top: 1.5pt"
    vAlign=top>
      <p class=MsoNormal style="text-align: center" align="center"><a
      href="http://www.arabidopsis.org/">Weed</a> </p></td>
    <td
    style="padding-right: 1.5pt; padding-left: 1.5pt; padding-bottom: 1.5pt; padding-top: 1.5pt"
    vAlign=top>
      <p class=MsoNormal style="text-align: center" align="center">5</p></td>
    <td
    style="padding-right: 1.5pt; padding-left: 1.5pt; padding-bottom: 1.5pt; padding-top: 1.5pt"
    vAlign=top>
      <p class=MsoNormal style="text-align: center"
    align="center">125,000,000</p></td></tr>
  <tr>
    <td
    style="padding-right: 1.5pt; padding-left: 1.5pt; padding-bottom: 1.5pt; padding-top: 1.5pt"
    vAlign=top>
      <p class=MsoNormal style="text-align: center" align="center"><a
      href="http://www.sanger.ac.uk/HGP/">Human</a> </p></td>
    <td
    style="padding-right: 1.5pt; padding-left: 1.5pt; padding-bottom: 1.5pt; padding-top: 1.5pt"
    vAlign=top>
      <p class=MsoNormal style="text-align: center" align="center">23</p></td>
    <td
    style="padding-right: 1.5pt; padding-left: 1.5pt; padding-bottom: 1.5pt; padding-top: 1.5pt"
    vAlign=top>
      <p class=MsoNormal style="text-align: center"
      align="center">3,000,000,000</p></td></tr></table>
<p
style="tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt">Note
that these sizes determine the upper bound of the genetic information in the
organism. </p>
<p
style="tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt">Determining
the four letter sequence for a given a DNA molecule is known as the <I>DNA
sequencing</I>. The first full genome for a bacterium was sequenced in 1995. The
yeast (<I>Saccharomyces cerevisiae</I>) genome was sequenced in 1997, worm
(nematode <I>Caenorhabditis elegans</I>) in 1999, fly (<I>Drosophila
melanogaster</I>) in 2000, and weed (<I>Arabidopsis thaliana</I> ) at 2001.
About 90% or so of the human genome was completed in 2001, this is known as the
<I>draft human genome</I>. </p>
<p
style="tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt">Sequencing
of the relatively small bacterial genomes has become routine and is largely done
by sequencing robots and completed by human researchers, the main problem being
the minimisation of costs per letter, and maximisation of the speed while
maintaining quality. Sequencing of larger genomes, like a human genome, is still
difficult, though most of the problems are computational. Sequencing robots are
able to sequence only relatively short stretches of DNA, which afterward have to
be assembled together by a computer using assembly algorithms. The main
difficulty is that genomes of higher eukaryotes (like humans) have many repeated
subsequences, which makes the assembly rather tricky, this means that
considerable human intervention is still needed in the final stage of sequencing
projects. </p>
<p
style="tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt">The
worlds largest public genome sequencing project is housed at the <a
href="http://www.sanger.ac.uk/">Sanger Institute</a> (the large building next to
the EBI). It is obvious, that because of the size of even the smallest bacterial
genomes, the genome sequencing would not make much sense unless the sequences
were stored in computer databases, allowing automatic search, comparison and
other analyses. All the public DNA sequences (including the completed genomes)
are stored in the <a href="http://www.ebi.ac.uk/embl/index.html">EMBL</a>
database (also known as EMBL-Bank), which is in fact a collaboration of three
databases EMBL in Europe, <a href="http://www.ncbi.nlm.nih.gov/">GenBank</a> in
the USA and <a href="http://www.ddbj.nig.ac.jp/">DDBJ</a> in Japan (each
database mirrors the others and they exchange data every 24 hours). </p>
<p
style="tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt">The
sequences contained in EMBL can be used as a resource to compare newly sequenced
stretches of DNA. If a similar sequence is already in the EMBL database we can
sometimes infer a biological function by analogy. Sequence comparison can be
done by dynamic programming algorithms, but heuristic comparison algorithms, for
instance <a href="http://www.ebi.ac.uk/fasta33/">FASTA</a> and <a
href="http://www.ebi.ac.uk/blast2/">BLAST</a> are more popular because of
speed.&nbsp;</p>
<p
style="tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt">Genomes
contain genes, most of which encode proteins. </p>
<h3
style="tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt">3.2
Mendel, peas and classical genetics</h3>
<p class=MsoNormal
style="margin-top: 5pt; tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt">Genetics
started in the 19th century without any knowledge about DNA. Gregor Mendel bred
peas in his garden and noticed that the inheritance sometimes follows clear
rules. For instance if we cross peas which have green and yellow seeds, while in
the next generation all seeds will be yellow, two generations later one quarter
of the seeds will be green, while three quarters will be yellow. To explain this
Mendel introduced the notion of <I>dominant</I> (yellow in our example) over the
<I>recessive</I> (green) properties and the notion of <I>gene</I> (though he did
not use this name yet) &#8211; the carrier of inheritance. This was before the
discovery of chromosomes. Later, but before the DNA structure was discovered, it
was correctly hypothesized that genes are located in chromosomes.</p>
<p class=MsoNormal
style="margin-top: 5pt; tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt">Not
all inheritable properties are so simple. Crossing of some plants with red and
white flowers can have a daughter generation with pink flowers. There is a lot
more fun stuff in this area of research, which is usually termed classical
genetics. This is a typical black box approach to biology - study the external
properties of a biological system and <I>guess</I> at a mechanism to explain
this. It become known only in the second half of the 20<sup>th</sup> century
that DNA is the carrier of genes, and that this information is passed from
generation to generation by DNA replication.</p><b><span
style="FONT-SIZE: 13.5pt; FONT-FAMILY: 'Times New Roman'; mso-fareast-font-family: 'Times New Roman'; mso-ansi-language: EN-GB; mso-fareast-language: EN-US; mso-bidi-language: AR-SA"><br
style="PAGE-BREAK-BEFORE: always; mso-special-character: line-break"
clear=all></span></b>
<h3
style="tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt"><![if !supportEmptyParas]><![endif]>&nbsp;<o:p></o:p></h3>
<h3
style="tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt">3.3
Genes and protein synthesis</h3>
<p class=MsoNormal
style="mso-margin-bottom-alt: auto; tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt">There
are many discussions between biologists to find a comprehensive definition of a
gene, which is not easy, if possible at all. For our purposes<br />&nbsp;</p>
<div align="center">
<table
style="width: 80%; mso-cellspacing: 1.5pt; mso-padding-alt: 0cm 0cm 0cm 0cm"
cellpadding="0" width="80%" border=1>
  
  <tr>
    <td
    style="padding-right: 0.75pt; padding-left: 0.75pt; padding-bottom: 0.75pt; padding-top: 0.75pt">
      <p class=MsoNormal><span style="COLOR: blue">A <I>gene</I> is a continuous
      stretch of a genomic DNA molecule, from which a complex molecular
      machinery can read information (encoded as a string of A, T, G, and C) and
      make a particular type of a protein or a few different
    proteins.</span></p></td></tr></table></div>
<p class=MsoNormal
style="mso-margin-bottom-alt: auto; tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt"><br />This
&#8220;definition&#8221; is not precise, and to better understand it we need to describe the
molecular machinery making proteins based on the information encoded in genes.
This process is called <I>protein synthesis</I> and has three essential stages:
(1) <I>transcription</I>, (2) <I>splicing</I>, and (3) <I>translation</I>.</p>
<p class=MsoNormal
style="margin-top: 5pt; tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt">1.
In <I>transcription</I> phase one strand of DNA molecule is copied into a
complementary pre mRNA (pre stands for preliminary and m for messenger) by the
protein complex RNA polymerase II (see section 2.2 and 2.4). In the process the
two-stranded DNA double helix is unwound and information is read only from one
strand (sometimes called the W-strand).&nbsp;</p>
<p class=MsoNormal
style="margin-top: 5pt; tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt">2.
<I>Splicing</I> removes some stretches of the pre mRNA, called <I>introns</I>,
the remaining sections called <I>exons</I> are then joined together. Note that
the removal of introns is a consequence of the way how eukaryote genomes are
organised.&nbsp;&nbsp; The genomic DNA that corresponds to the coding part of
genes is not continuous, but consists of exons and introns. Exons are the part
of the gene that code for proteins and they are interspersed with non coding
introns which must be removed by splicing. The number and&nbsp; size of introns
and exons differs considerably between genes and also between species. Only very
few genes in yeast have introns, while&nbsp; for human threre are about 4
introns per gene on average, and the average size of exons is 150 bp and just
above 3400 bp for introns. Prokaryote genes do not have introns and the splicing
step is not present. The result of splicing is mRNA. Many eukaryote genes are
known to have different alternative splice variants, i.e. the same pre-mRNA
producing different mRNAs, known as <I>alternative splicing</I>.</p>
<p class=MsoNormal
style="margin-left: 18pt; tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt">&nbsp;</p>
<p class=MsoNormal
style="margin-left: 36pt; tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt"><img
id=_x0000_i1040 height=209 src="biology_intro_files/exintrons.gif" width=340
border="0">(picture taken from&nbsp; <a
href="http://gened.emc.maricopa.edu/bio/bio181/BIOBK/BioBookTOC.html">On-Line
Biology Book</a> )<o:p></o:p></p>
<p class=MsoNormal
style="tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt"><U1:p></U1:p>3.
<I>Translation</I> is the process of making proteins by joining together amino
acids in order encoded in the mRNA. The order of the amino acids is determined
by 3 adjacent nucleotides (triplets) in the DNA. This is known as the <I><a
href="biology_intro_files/code.html">triplet or genetic code</a>
</I>. Each triplet is called a <I>codon</I> and codes for one amino acid. As
there are 64 codons and only 20 amino acids the code is redundant, for example
histidine is encoded by CAT and CAC. &nbsp;In cytoplasm the mRNA forms a complex
with <I>ribosomes</I>, which are large complexes of proteins and RNA molecules.
The precise interactions and functions of all protein in ribosomes are not yet
fully understood. </p>
<p class=MsoNormal
style="tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt"><br /><img
id=_x0000_i1065 height=239 src="biology_intro_files/polysome.gif" width=400
border="0">(picture taken from&nbsp; <a
href="http://gened.emc.maricopa.edu/bio/bio181/BIOBK/BioBookTOC.html">On-Line
Biology Book</a> ) <br />Different transfer or <a
href="biology_intro_files/tRNA.htm">tRNA</a> molecules
each carries one specific amino acid to the ribosome and specifically recognises
one codon on the mRNA. The amino acid carried by the tRNA is added to the
nascent (growing) protein. The <a
href="http://gened.emc.maricopa.edu/bio/bio181/BIOBK/BioBookPROTSYn.html#Protein%20Synthesis">translation
is a complex process</a> and not all the details are understood. Luckily most of
these details are not crucial for understanding of bioinformatics. What is
crucial however is to realise that there is nothing magical about proteins
synthesis. <br /><span style="mso-spacerun: yes">&nbsp;</span></p>
<p class=MsoNormal
style="margin-bottom: 12pt; tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt">The
end of translation is the final part of gene expression and the final product is
a protein, the sequence of which corresponds to the sequence encoded by the
mRNA. Proteins can be <I>post-translationally modified</I> e.g., by adding of
sugars or cleavage (chopping), and this affects their location and function.</p>
<p class=MsoNormal
style="tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt">Biologists
used to believe in paradigm - 'one gene - one protein'. Now this is known not to
be true - due to <a
href="biology_intro_files/gene_altSplice.html">alternative
splicing</a> and post-translational modifications one gene can produce a variety
of proteins. There are also genes that do not encode proteins but encode RNA
(for instance tRNA and ribosomal RNA).&nbsp;<o:p></o:p></p>
  <h3
style="tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt"><U1:p></U1:p>3.4
Gene prediction, counting, annotation, ENSEMBL and UniProtKB/TrEMBL</h3>
<p class=MsoNormal
style="margin-bottom: 12pt; tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt">It
is an interesting question: given the genomic DNA sequence, can we tell where
the genes are? The answer is that we can, although the accuracy of such
predictions is not very high. Most of the knowledge to do these predictions
comes from experimentally identified genes. There are about 6700 experimentally
confirmed genes in the human genome (to be more accurate - 6700 experimentally
verfied cDNAs). Gene prediction is an important problem for computational
biology and there are various algorithms that do gene prediction using known
genes as a training data set. A popular algorithmic technique used in gene
prediction are hidden Markov models (HMMs).</p>
<p class=MsoNormal
style="tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt">Even
if we knew where the genes are in the genome, it is not entirely obvious how to
count them. Due to the existence of overlapping genes and splice variants it is
difficult to define which parts of the DNA should be regarded as the same or
several different genes. Nevertheless, for practical purposes (allowing for some
'experimental error') we can count how many genes an organism has. Some of the
results of counting of predicted genes have turned out to be quite
surprising.</p>
<p class=MsoNormal
style="tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt"><br />&nbsp;
</p><span
style="FONT-SIZE: 12pt; FONT-FAMILY: 'Times New Roman'; mso-fareast-font-family: 'Times New Roman'; mso-ansi-language: EN-GB; mso-fareast-language: EN-US; mso-bidi-language: AR-SA"><br
style="PAGE-BREAK-BEFORE: always; mso-special-character: line-break"
clear=all></span>
<p class=MsoNormal
style="tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt"><![if !supportEmptyParas]><![endif]>&nbsp;<o:p></o:p></p>
<table
style="width: 80%; mso-cellspacing: 1.5pt; mso-padding-alt: 0cm 0cm 0cm 0cm"
cellpadding="0" width="80%" border=1>
  
  <tr>
    <td
    style="padding-right: 0.75pt; padding-left: 0.75pt; padding-bottom: 0.75pt; padding-top: 0.75pt">
      <p class=MsoNormal>Organism</p></td>
    <td
    style="padding-right: 0.75pt; padding-left: 0.75pt; padding-bottom: 0.75pt; padding-top: 0.75pt">
      <p class=MsoNormal>The number of </p>
      <p class=MsoNormal>predicted genes</p></td>
    <td
    style="padding-right: 0.75pt; padding-left: 0.75pt; padding-bottom: 0.75pt; padding-top: 0.75pt">
      <p class=MsoNormal>Part of the genome that&nbsp;<br />encodes proteins
      (exons) </p></td></tr>
  <tr>
    <td
    style="padding-right: 0.75pt; padding-left: 0.75pt; padding-bottom: 0.75pt; padding-top: 0.75pt">
      <p class=MsoNormal>E.Coli (bacteria)</p></td>
    <td
    style="padding-right: 0.75pt; padding-left: 0.75pt; padding-bottom: 0.75pt; padding-top: 0.75pt"><pre><span style="mso-spacerun: yes">&nbsp;&nbsp;&nbsp;&nbsp; </span>5000</pre></td>
    <td
    style="padding-right: 0.75pt; padding-left: 0.75pt; padding-bottom: 0.75pt; padding-top: 0.75pt"><pre><span style="mso-spacerun: yes">&nbsp;&nbsp;&nbsp; </span>90%</pre></td></tr>
  <tr>
    <td
    style="padding-right: 0.75pt; padding-left: 0.75pt; padding-bottom: 0.75pt; padding-top: 0.75pt">
      <p class=MsoNormal
      style="tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt">Yeast</p></td>
    <td
    style="padding-right: 0.75pt; padding-left: 0.75pt; padding-bottom: 0.75pt; padding-top: 0.75pt"><pre><span style="mso-spacerun: yes">&nbsp;&nbsp;&nbsp;&nbsp; </span>6000</pre></td>
    <td
    style="padding-right: 0.75pt; padding-left: 0.75pt; padding-bottom: 0.75pt; padding-top: 0.75pt"><pre><span style="mso-spacerun: yes">&nbsp;&nbsp;&nbsp; </span>70%</pre></td></tr>
  <tr>
    <td
    style="padding-right: 0.75pt; padding-left: 0.75pt; padding-bottom: 0.75pt; padding-top: 0.75pt">
      <p class=MsoNormal
      style="tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt"><a
      href="biology_intro_files/celegans.html">Worm</a>
      </p></td>
    <td
    style="padding-right: 0.75pt; padding-left: 0.75pt; padding-bottom: 0.75pt; padding-top: 0.75pt"><pre><span style="mso-spacerun: yes">&nbsp;&nbsp; </span>18,000</pre></td>
    <td
    style="padding-right: 0.75pt; padding-left: 0.75pt; padding-bottom: 0.75pt; padding-top: 0.75pt"><pre><span style="mso-spacerun: yes">&nbsp;&nbsp;&nbsp; </span>27%</pre></td></tr>
  <tr>
    <td
    style="padding-right: 0.75pt; padding-left: 0.75pt; padding-bottom: 0.75pt; padding-top: 0.75pt">
      <p class=MsoNormal
      style="tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt"><a
      href="biology_intro_files/drosophyla.html">Fly</a>
      </p></td>
    <td
    style="padding-right: 0.75pt; padding-left: 0.75pt; padding-bottom: 0.75pt; padding-top: 0.75pt"><pre><span style="mso-spacerun: yes">&nbsp;&nbsp; </span>14,000</pre></td>
    <td
    style="padding-right: 0.75pt; padding-left: 0.75pt; padding-bottom: 0.75pt; padding-top: 0.75pt"><pre><span style="mso-spacerun: yes">&nbsp;&nbsp;&nbsp; </span>20%</pre></td></tr>
  <tr>
    <td
    style="padding-right: 0.75pt; padding-left: 0.75pt; padding-bottom: 0.75pt; padding-top: 0.75pt">
      <p class=MsoNormal
      style="tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt"><a
      href="biology_intro_files/arabidopsis.html">Weed</a>
      </p></td>
    <td
    style="padding-right: 0.75pt; padding-left: 0.75pt; padding-bottom: 0.75pt; padding-top: 0.75pt"><pre><span style="mso-spacerun: yes">&nbsp;&nbsp; </span>25,500<U1:p></U1:p><o:p></o:p></pre></td>
    <td
    style="padding-right: 0.75pt; padding-left: 0.75pt; padding-bottom: 0.75pt; padding-top: 0.75pt"><pre><U1:p></U1:p><span style="mso-spacerun: yes">&nbsp;&nbsp;&nbsp; </span>20%</pre></td></tr><U6:p></U6:p>
  <tr>
    <td
    style="padding-right: 0.75pt; padding-left: 0.75pt; padding-bottom: 0.75pt; padding-top: 0.75pt">
      <p class=MsoNormal
      style="tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt"><a
      href="biology_intro_files/homosapiens.html">Human</a>
      </p></td>
    <td
    style="padding-right: 0.75pt; padding-left: 0.75pt; padding-bottom: 0.75pt; padding-top: 0.75pt"><pre><span style="mso-spacerun: yes">&nbsp;&nbsp; </span>30,000</pre></td>
    <td
    style="padding-right: 0.75pt; padding-left: 0.75pt; padding-bottom: 0.75pt; padding-top: 0.75pt"><pre><span style="mso-spacerun: yes">&nbsp;&nbsp;&nbsp; </span>&lt; 5%</pre></td></tr></table><U6:p></U6:p>
<p class=MsoNormal
style="margin-top: 5pt; tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt"><br />One
of the surprises is the relatively small number of genes in a human genome in
comparison to worm. Before most of the human genome sequencing was accomplished,
it was estimated that there should be about 100,000 genes in a human. In fact
some experts still think that there must be at least 40,000 - 50,000 genes in
the human genome, and that 30,000 just reflects the unreliability of<I> in
silico </I>(i.e., computational) gene prediction. Still, it seems that there is
no simple correlation between the intuitive (not well-defined) complexity of an
organism and the number of genes in its genome (for instance, intuitively <a
href="http://flybase.bio.indiana.edu:7081/images/lk/Drosophilidae/Drosophila_melanogaster_m/index.html">fly</a>
is more complex organism than <a href="http://www.tigr.org/tdb/cegi/">worm</a>
). One reason for the low number of genes in the human genome may be that there
are more splice variants per gene in humans, though this has yet to be proved
(otherwise human vanity may have to suffer).</p>
<p class=MsoNormal
style="margin: 5pt 0cm 12pt; tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt">The
presence of 95% of non-coding DNA in the human genome (sometimes called the junk
DNA) remains a mystery. There are several hypotheses explaining this, but none
is generally accepted. One controversial hypothesis (promoted by Richard
Dawkins) is based on the idea of so-called selfish DNA. It states that the DNA
is the basic element for natural selection, implying that DNA tries to propagate
(multiply and amplify) itself, while the cells and organisms are vehicles to
achieve this.&nbsp; </p>
<p class=MsoNormal
style="margin-bottom: 12pt; tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt">Automated
gene prediction has been critical in making sense of the unfinished or draft
data from genome projects. Sequence data is assembled into larger pieces
(contigs), related to known genetic markers on chromosomes to order them and the
gene predictions can then be placed on the unfinished sequence. The <a
href="http://www.ensembl.org/">ENSEMBL</a> project offers annotation for gene
prediction in eukaryotes. ENSEMBL is a joint project between the EBI and the
Sanger Institute.</p>
  <p class=MsoNormal
style="mso-margin-bottom-alt: auto; tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt">Currently
most new entries to UniProtKB/Swiss-Prot database come from UniProtKB/TrEMBL, which are automatically
translated DNA sequences of predicted genes in EMBL database into protein
sequences.</p>
<h3
style="tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt">3.5
Genome similarity, SNPs and comparative genomics</h3>
<p class=MsoNormal
style="margin-bottom: 12pt; tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt">Those
who are learning about genome sequencing for the first time often ask the
question - whose genome is being sequenced? Is it a particular individual? This
is not an unreasonable question, in fact several anonymous samples were
collected and pooled for the human genome project. But this does not really
matter as all human genomes are deemed to be roughly 99.9% equivalent and on
average one in a thousand nucleotides are different in genomes of two different
individuals. Therefore we can talk in terms of the consensus human genome.</p>
<p class=MsoNormal
style="tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt">The
0.1% difference is exploited in DNA fingerprinting used in criminology.
Variations in non-coding parts of the genome are analysed to produce patterns
that can reliably distinguish individuals, an exception here is identical twins
which are much harder to distinguish with DNA fingerprinting.<o:p></o:p></p>
<p class=MsoNormal
style="tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt"><U1:p></U1:p>Particularly
important variations in individual genomes are the <I>single nucleotide
polymorphisms</I> or SNPs, which can occur both in coding and non-coding parts
of the genome. SNPs are DNA sequence variations which occur when a single base
(A,C,G, or T) is altered so that different individuals may have different
letters in these positions. Particular nucleotides in SNP positions within or
close to genes can influence the gene's protein product. Some 'abnormal' protein
variants (mutant variants) are the cause of genetic diseases. SNPs may be
responsible for many inheritable differences between individuals, and SNP
variation may indicate the predisposition to a genetic disease. For example
there is evidence that certain combinations of SNPs occur in individuals with
Alzheimers disease. SNP analysis is therefore important for diagnostics and a <a
href="http://www.ebi.ac.uk/mutations/">SNP database</a> is being built with the
participation of the EBI. SNP analysis can also be used in population genetics,
as some SNPs vary in frequency between populations. There are about 3 million
SNPs collected in public SNP databases.&nbsp;</p>
<p class=MsoNormal
style="tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt"><![if !supportEmptyParas]><![endif]>&nbsp;<o:p></o:p></p>
<p class=MsoNormal
style="mso-margin-bottom-alt: auto; tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt"><U1:p></U1:p>The
sequencing projects have revealed that the genomes of what look like very
different organisms may be quite similar. An example of this is similarity
between yeast and worm described in the next section. It is estimated that the
difference between human and chimpanzee genomes is only 1-3%, while between
human and mouse only about 5% - 15%, depending on how the similarity is defined
and measured (before the sequencing of chimp and mouse are well under way, these
are only estimates). These similarities indicate close evolutionary
relationships between these mammalian organisms. It is possible to build a
<I>phylogenetic tree</I> of the evolution of proteins, genes and organisms based
on sequence comparisons.</p>
<h2
style="tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt">4.&nbsp;<a
name=Functional></a> Functional genomics</h2>
<p class=MsoNormal
style="mso-margin-bottom-alt: auto; tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt">Functional
genomics can be roughly defined as using the emerging knowledge about genomes to
understand the gene and their product functions and interactions, and most
importantly of all, how all this makes organisms to function the way they
do.</p>
<h3
style="tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt">4.1
Gene functions and GO</h3>
<p class=MsoNormal
style="tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt">As
we already mentioned, proteins play a variety of roles in a cell. Biologists
think that there is likely to be a limited universe of genes and their
respective proteins, from the functional point of view, many of which are
present in most or all genomes. For instance, about 12% of the ~18 000 worm
genes encode proteins whose biological roles could be inferred from their
sequence similarity to yeast genes, and vice versa - almost a third of the ~6000
yeast genes have functional equivalents in worm genes. The recognition of the
limited universe of genes has prompted unification of biology under the name <a
href="http://www.geneontology.org/">Gene Ontology</a> or GO.</p>
<p class=MsoNormal
style="tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt"><![if !supportEmptyParas]><![endif]>&nbsp;<o:p></o:p></p>
  <p class=MsoNormal
style="tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt"><U1:p></U1:p>The
aim of the GO project is the development of a dynamic controlled vocabulary for
description of gene function and localisation. GO works at three levels for each
gene product: <I>molecular function</I>, <I>&nbsp;biological process</I> and
<I>cellular component</I>. The vocabulary is organised in three independent
directed acyclic graphs (DAG's) characterising the genes at each of these three
levels. The use of GO terms allows automated searching of gene function in
databases. Proteins in UniProtKB/Swiss-Prot and the Interpro databases (another project at
the EBI that classifies proteins by a variety of computational methods) are
currently being assigned <a href="http://www.ebi.ac.uk/ego/">GO terms</a> .</p>
<h3
style="tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt">4.2
Protein abundance in a cell</h3>
  <p class=MsoNormal
style="margin-bottom: 12pt; tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt">Proteins
in a cell are synthesised from genes and their life cycle can be roughly
described as <I>synthesis</I>, <I>functionality</I> and <I>degradation</I>.
Nobody really knows how many different proteins are synthesized from the
estimated 30,000 genes in a human cell. Due to alternative splicing and
post-translational modifications this number apparently is much higher than the
number of genes. Evidently, not all the proteins have to be present in a given
cell at a given moment.</p>
<p class=MsoNormal
style="margin-bottom: 12pt; tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt">The
protein abundance may depend on many factors such as whether the respective gene
is <I>expressed</I> (i.e., is actively transcribed) or not, how intensively (how
fast) it is expressed, whether and how fast it is spliced, translated and
modified, how long a half-life the mRNA and the protein have, and whether it is
actively degraded at a given moment. Direct experimental studies of protein
abundance are technically difficult at present. However, thanks to the
microarray technology (see next section), it is possible to measure the mRNA
abundance (gene expression) for tens of thousands of genes in parallel in a
single experiment. The correlation between gene expression and the presence of
the respective proteins in the cell is not straightforward, still, in many cases
some estimates about the proteins can be made from gene expression.</p>
<p class=MsoNormal
style="mso-margin-bottom-alt: auto; tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt">There
is evidence that for most of the 320 different cell types in a human body, only
about 10%-20% of the genes are expressed in any particular state or
developmental stage. These numbers are not very reliable as the studies are
difficult and some gene products may have very low abundance levels, which may
be difficult to detect experimentally.</p>
<p class=MsoNormal
style="mso-margin-bottom-alt: auto; tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt"><br />Another
question related to protein abundance in a cell is which proteins interact with
which at some point in cell's life. This is known as <I>protein-protein
interaction</I> networks. There is a plan to build a protein-protein interaction
database at the EBI and there is work going on at the EBI to predict such
interactions by computational methods. </p>
<h3
style="tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt">4.3
Gene regulation and networks</h3>
  <p class=MsoNormal
style="margin-bottom: 12pt; tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt">Another
important and interesting question in biology is how gene expression is switched
on and off, i.e., how genes are regulated. Since almost all cells in a
particular organism have an identical genome (section 3), differences in gene
expression and not the genome content are responsible for cell differentiation
(how different cell types develop from a fertilised egg) during the life of the
organism.</p>
<p class=MsoNormal
style="margin-bottom: 12pt; tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt">Gene
regulation in eukaryotes, is not well understood, but there is evidence that an
important role is played by a type of proteins called <I>transcription
factors</I>. The transcription factors can attach (bind) to specific parts of
the DNA, called transcription factor <I>binding sites</I> (i.e., specific,
relatively short combinations of A, T, C or G), which are located in so-called
<I>promoter </I>regions. Specific promoters are associated with particular genes
and are generally not too far from the respective genes, though some regulatory
effects can be located as far as 30,000 bases away, which makes the definition
of the promoter difficult. .</p>
<p class=MsoNormal
style="tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt">Transcription
factors control gene expression by binding the gene's promoter and either
activating (switching on) the gene's transcription, or repressing it (switching
it off). Transcription factors are gene products themselves, and therefore in
turn can be controlled by other transcription factors. Transcription factors can
control many genes, and some (probably most) genes are controlled by
combinations of transcription factors. Feedback loops are possible. Therefore we
can talk about <I>gene regulation networks</I> . Understanding, describing and
modelling such gene regulation networks are one of the most challenging problems
in functional genomics.</p>
<p class=MsoNormal
style="tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt"><br />There
are over 200 known transcription factors in yeast, over 600 in worm and fly, and
over 1500 in weed, but the real numbers are probably higher, since more than
half of the predicted genes in these organism have unknown function and there
are transcription factors likely to be found among the unknown genes. In
addition, transcription factors are not the only proteins participating in gene
regulation - proteins affecting chromatine structure are another example, and it
is know that at least some regulation is happening in the translation
stage.&nbsp; Microarrays and computational methods are playing a major role in
attempts to reverse engineer gene networks from various observations. Note that
in reality the gene regulation is likely to be a stochastic and not a
deterministic process.&nbsp;</p>
<p class=MsoNormal
style="tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt"><br />Traditionally
molecular biology has followed so-called reductionist approach mostly
concentrating on a study of a single or very few genes in any particular
research project. There is a proverb &#8211; &#8216;one gene at a time&#8217; or &#8216;one post-doc,
one gene&#8217;. With genomes being sequenced, this is now changing into so-called
systems approach. We can begin asking questions such as how many genes are
expressed in different cell types, which genes are expressed in all cell types,
what are the functional roles of these genes, how big is the gene function
universe, how many genes are needed for life, how it can be that a worm has more
genes than a fly, and the human only a bit more than a worm, and, of course, we
can always revisit the question of the <a
href="biology_intro_files/meaning_of_life.htm">meaning of
life.</a> </p>
<h2
style="tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt">5&nbsp;<a
name=Microarrays></a> Microarrays and gene expression databases</h2>
<p class=MsoNormal
style="margin-top: 5pt; tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt">Microarray
technology makes use of the sequence resources created by the genome projects
and other sequencing efforts to answer the question, what genes are expressed in
a particular cell type of an organism, at a particular time, under particular
conditions. For instance, they allow comparison of gene expression between
normal and diseased (e.g., cancerous) cells. There are several names for this
technology - DNA microarrays, DNA arrays, DNA chips, gene chips, others.
Sometimes a distinction is made between these names but in fact they are all
synonyms as there are no standard definitions for which type of microarray
technology should be called by which name.</p>
<h3
style="tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt">5.1
Microarray technology and applications</h3>
<p class=MsoBodyText
style="text-align: left; tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt"
align="left"><span lang=EN-US>Microarrays exploit the preferential binding of
complementary single-stranded nucleic acid sequences (see section 2.3-2.4). A
microarray is typically a glass (or some other material) slide, on to which DNA
molecules are attached at fixed locations (spots). There may be tens of
thousands of spots on an array, each containing a huge number of identical DNA
molecules (or fragments of identical molecules), of lengths from twenty to
hundreds of nucleotides.&nbsp; (According to quick napkin calculations by
Wilhelm Ansorge and John Quackenbush in Schnookeloch in Heidelberg on 4 October,
2001, the number of DNA molecules in a microarry <a
href="biology_intro_files/spot.htm">spot</a> is
10<sup>7</sup>-10<sup>8</sup>). For gene expression studies, each of these
molecules ideally should identify one gene or one exon in the genome, however,
in practice this is not always so simple and may not even be generally possible
due to families of similar genes in a genome. Microarrays that contain all of
the about 6000 genes of the yeast genome have been available since 1997. The
spots are either printed on the microarrays by a robot, or synthesized by
photo-lithography (similarly as in computer chip productions) or by ink-jet
printing.</span></p>
<table
style="mso-cellspacing: 0cm; mso-padding-alt: 0cm 0cm 0cm 0cm; mso-table-anchor-vertical: paragraph; mso-table-anchor-horizontal: column; mso-table-left: left"
cellspacing="0" cellpadding="0" align="left" border="0">
  
  <tr style="height: 14.25pt">
    <td
    style="padding-right: 0cm; padding-left: 0cm; padding-bottom: 0cm; width: 7.2pt; padding-top: 0cm; height: 14.25pt"
    width=10>
      <p class=MsoNormal>&nbsp;</p></td></tr>
  <tr>
    <td
    style="padding-right: 0cm; padding-left: 0cm; padding-bottom: 0cm; padding-top: 0cm">
      <p class=MsoNormal>&nbsp;</p></td>
    <td
    style="padding-right: 0cm; padding-left: 0cm; padding-bottom: 0cm; padding-top: 0cm">
      <p class=MsoNormal><img id=_x0000_i1086 height=250 src="biology_intro_files/image002.jpg" width=229
      border="0" u2:shapes="_x0000_s1026"></p></td></tr></table>
<p class=MsoNormal
style="tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt"><br /><U1:p></U1:p>&nbsp;
</p>
<p class=MsoBodyText2
style="tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt"><span
lang=EN-US style="mso-ansi-language: EN-US">An illuminated microarray
(enlarged). A typical dimension of such an array is about 1 inch or less, the
spot diameter is of the order of 0.1 mm, for some microarray types can be even
smaller.<U1:p></U1:p></span><o:p></o:p></p>
<p class=MsoBodyText
style="tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt"><span
lang=EN-US><U1:p></U1:p></span></p>
<p class=MsoBodyText
style="tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt"><span
lang=EN-US></span>&nbsp;</p>
<p class=MsoBodyText
style="tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt"><span
lang=EN-US></span>&nbsp;</p>
<p class=MsoBodyText
style="tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt"><span
lang=EN-US></span>&nbsp;</p>
<p class=MsoBodyText
style="tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt"><span
lang=EN-US></span>&nbsp;</p>
<p class=MsoBodyText
style="tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt"><span
lang=EN-US></span>&nbsp;</p>
<p class=MsoBodyText
style="tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt"><span
lang=EN-US></span>&nbsp;</p>
<p class=MsoBodyText
style="text-align: left; tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt"
align="left"><span lang=EN-US><br />There are different ways how microarrays can be
used to measure the gene expression levels. One of the most popular micorarray
applications allows to compare gene expression levels in two different samples,
e.g., the same cell type in a healthy and diseased state (see figure
below).<o:p></o:p></span></p>
<p class=MsoNormal
style="margin-bottom: 6pt; text-align: justify; tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt"><U2:SHAPE
id=_x0000_i1079 style="width: 5in; height: 270pt" u1:ole=""
type="#_x0000_t75"><U2:IMAGEDATA _x002f_="" u1:title=""
src="image003.wmz"></U2:IMAGEDATA></U2:SHAPE><img id=_x0000_i1101 height=360
src="biology_intro_files/image004.gif" width=480 border="0" u2:shapes="_x0000_i1079"><o:p></o:p></p><U1:p></U1:p>
  <p class=MsoBodyText
style="tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt"><span
lang=EN-US>The total mRNA from the cells in two different conditions is
extracted and labeled with two different fluorescent labels: for example a
green dye for cells at condition 1 and a red dye for cells at condition 2 (to be
more accurate, the labeling is typically done by synthesising single stranded
DNA's that are complementary to the extracted mRNA by a enzyme called reverse
transcriptase). Both extracts are washed over the microarray. Labeled gene
products from the extracts hybridise to their complementary sequences in the
spots due to the preferential binding - complementary single stranded nucleic
acid sequences tend to attract to each other and the longer the complimentary
parts, the stronger the attraction (see section 2.3 and 2.4). </span></p>
<p class=MsoNormal
style="margin-bottom: 6pt; tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt">The
dyes enable the amount of sample bound to a spot to be measured by the level of
fluorescence emitted when it is excited by a laser. If the RNA from the sample
in condition 1 is in abundance, the spot will be green, if the RNA from the
sample in condition 2 is in abundance, it will be red. If both are equal, the
spot will be yellow, while if neither are present it will not fluoresce and
appear black. Thus, from the fluorescence intensities and colours for each spot,
the relative expression levels of the genes in both samples can be estimated.
</p>
<p class=MsoNormal
style="margin-bottom: 6pt; text-align: justify; tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt"><img
id=_x0000_i1088 height=360 src="biology_intro_files/image006.gif" width=480 border="0"
u2:shapes="_x0000_i1080"></p>
<p class=MsoNormal
style="margin-bottom: 6pt; tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt"><br />The
raw data that are produced from microarray experiments are the hybridised
microarray images. To obtain information about gene expression levels, these
images should be analysed, each spot on the array identified, its intensity
measured and compared to the background. This is called image quantitation.
<U1:p></U1:p><o:p></o:p></p>
<p class=MsoBodyText2
style="text-align: left; tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt"
align="left"><U7:SHAPE id=_x0000_i1080 style="width: 5in; height: 270pt"
type="#_x0000_t75" u6:ole=""><U7:IMAGEDATA src="image005.wmz" _x002=""
u6:title=""></U7:IMAGEDATA></U7:SHAPE><span style="FONT-SIZE: 12pt">Image
quantiation is done by image analysis software. To obtain the final gene
expression matrix from spot quantiations, all the quantities related to some
gene (either on the same array or on arrays measuring the same conditions in
repeated experiments) have to be combined and the entire matrix has to be scaled
to make different arrays comparable. <o:p></o:p></span></p>
<p class=MsoBodyText2
style="text-align: left; tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt"
align="left">Gene expression monitoring is not the only microarray application,
another one is SNP detection. <br />&nbsp; </p>
<h3
style="tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt">5.2
ArrayExpress</h3>
  <p class=MsoNormal
style="margin-bottom: 6pt; tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt"><span
lang=EN-US style="mso-ansi-language: EN-US">Microarray's are already producing
massive amounts of data. These data, like genome sequence data, can help us to
gain insights into underlying biological processes only if they are carefully
recorded and stored in databases, where they can be queried, compared and
analysed by different computer software programs. The EBI is currently
establishing a public archive for microarray gene expression data <a
href="http://www.ebi.ac.uk/arrayexpress">ArrayExpress</a>, analogous to
EMBL-bank for DNA sequence data. In many respects gene expression databases are
inherently more complex than sequence databases (this does not mean that
developing, maintaining and curating the sequence databases are any less
challenging).<U1:p></U1:p></span></p>
<p class=MsoNormal
style="margin-bottom: 6pt; tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt"><span
lang=EN-US style="mso-ansi-language: EN-US">Conceptually,&nbsp; a gene
expression database can be regarded as consisting of three parts &#8211; the gene
expression data matrix, gene annotation and sample
annotation.<U1:p></U1:p><o:p></o:p></span></p>
<p class=MsoNormal
style="margin: 6pt 0cm; text-align: center; tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt"
align="center"><img id=_x0000_i1083 height=360
src="biology_intro_files/image008.gif" width=480 border="0"
u7:shapes="_x0000_i1081"></p>
<p class=MsoBodyText
style="text-align: left; tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt"
align="left"><span lang=EN-US>Gene expression data have meaning only in the
context of the particular biological sample and the exact conditions under which
the samples were taken. For instance, if we are interested in finding out how
different cell types react to treatments with various chemical compounds, we
must record unambiguous information about the cell types and compounds used in
the experiments. EBI is participating in an effort to develop <a
href="http://www.cbil.upenn.edu/Ontology/MGED_ontology.html">ontologies for
sample annotation</a> , this is analogous to gene ontology for gene description.
</span></p>
<p class=MsoBodyText
style="text-align: left; tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt"
align="left"><span lang=EN-US>Gene annotation&nbsp; can be taken care to some
extent of by links to sequence databases, unfortunately complicated many-to-many
relationships between genes in the gene expression matrix and the features
(spots) on the array makes it necessary to provide a full and detailed
description of each feature on the array, as one gene can relate to several
features on the array. The lack of standards in gene naming is another
difficulty - a table relating each array feature present in the database to the
list of all synonymous names of the respective gene is an essential part of a
gene expression database.&nbsp; </span></p>
<p class=MsoBodyText
style="text-align: left; tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt"
align="left"><span lang=EN-US>The microarray technology is still rapidly
developing, therefore it is natural that currently there are no established
standards for microarray experiments and how the raw data should be processed.
There are also no standard measurement units for gene expression levels. In the
lack of such standards the information about how exactly the gene expression
data matrix was obtained should be kept in the database, if the data are to be
properly interpreted later. This complicates the <a
href="biology_intro_files/AE-complete.png">data object
model</a> enormously. </span></p>
<p class=MsoBodyText
style="text-align: left; tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt"
align="left"><span lang=EN-US>ArrayExpress is storing all this information, the
details of which is called Minimum Information About a Microarray Experiment (<a
href="http://www.mged.org/Annotations-wg/index.html"> MIAME</a> ) defined by the
Microarray Gene Expression Database (<a href="http://www.mged.org/"> MGED</a> )
consortium. MGED is a grass roots movement that was founded at a <a
href="http://www.mged.org/MGED14111999/index.html">meeting at the EBI</a> in
1999, is supported by most of the important players in the microarray community,
and has evolved far beyond the EBI. </span></p>
<p class=MsoBodyText
style="text-align: left; tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt"
align="left"><span lang=EN-US>Another archive for gene expression data <a
href="http://www.ncbi.nlm.nih.gov/geo/">GEO</a> is being developed at NCBI in
the US. DDBJ in Japan also have plans. All three groups face similar problems
and are involved in MGED to some degree. A common data exchange format <a
href="http://cgi.omg.org/cgi-bin/doc?lifesci/01-08-01">MAGE-ML</a> is being
developed in collaboration between MGED (with active participation of the EBI)
and some major microarray companies. <br />&nbsp; </span></p>
<p class=MsoBodyText
style="text-align: left; tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt"
align="left"><span
lang=EN-US><![if !supportEmptyParas]><![endif]>&nbsp;<o:p></o:p></span></p>
<h3
style="tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt">6.3
Gene expression data analysis and Expression Profiler</h3>
<p class=MsoNormal
style="margin-bottom: 6pt; tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt"><span
lang=EN-US style="mso-ansi-language: EN-US">Capturing and storage of microarray
data is not an end in itself. The amounts of data from even a single microarray
experiment are so large, that software tools have to be used to make any sense
out of it. Clustering and class prediction are typical methods currently used in
gene expression data analysis (see <a
href="http://www.elsevier.nl/febs/118/19/21/index.htt">Microarray Data
Analysis</a> ). One of the popular gene expression data analysis tools is <a
href="http://ep.ebi.ac.uk/">Expression Profiler</a> , developed at the EBI. The
<a href="http://www.ebi.ac.uk/microarray">Functional Genomics Team</a> at the
EBI is actively working in many microarray data analysis areas using this and
other tools. <br />&nbsp;<U1:p></U1:p></span></p>
<p class=MsoNormal
style="margin-bottom: 6pt; tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt"><span
lang=EN-US style="mso-ansi-language: EN-US">An example of such research is an
approach to reverse engineering of gene regulatory networks, which is based on
the hypothesis that genes that have similar expression profiles (i.e., similar
rows in the gene expression matrix) should also have similar regulation
mechanisms as there must be a reason why their expression is similar under a
variety of conditions. Therefore, if we cluster the genes by similarities in
their expression profiles and take sets of promoter sequences from genes in such
clusters, some of these sets of sequences may contain a &#8216;signal&#8217; as a specific
sequence pattern such as a particular substring, which is relevant to regulation
of these genes (see <a
href="biology_intro_files/ISMB2000.pdf">Mining for
putative regulatory elements in the yeast genome using gene expression data</a>
).<U1:p></U1:p></span></p>
<p
style="tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt"><span
lang=EN-US style="mso-ansi-language: EN-US">There is much more <a
href="http://www.ebi.ac.uk/Groups/index.html">exciting stuff going on at the
EBI</a> , to join click <a
href="http://www.ebi.ac.uk/Information/Jobs/jobs.html">here</a>
.<U1:p></U1:p></span></p>
<h3
style="text-align: justify; tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt"><span
lang=EN-US
style="mso-ansi-language: EN-US">Acknowledgements<U1:p></U1:p></span></h3>
<ul style="margin-top: 0cm" type=disc>
  <li class=MsoNormal
  style="tab-stops: list 36.0pt; mso-list: l0 level1 lfo9"><span lang=EN-US
  style="mso-ansi-language: EN-US">Wilhelm Ansorge (EMBL)<U1:p></U1:p> </span>
  <li class=MsoNormal
  style="tab-stops: list 36.0pt; mso-list: l0 level1 lfo9"><span lang=EN-US
  style="mso-ansi-language: EN-US">Ewan Birney<U1:p></U1:p> </span>
  <li class=MsoNormal
  style="tab-stops: list 36.0pt; mso-list: l0 level1 lfo9"><span lang=EN-US
  style="FONT-SIZE: 7pt; mso-ansi-language: EN-US">&nbsp;</span><span lang=EN-US
  style="mso-ansi-language: EN-US">Gunars Brazma (University of
  Latvia)<U1:p></U1:p> </span>
  <li class=MsoNormal
  style="tab-stops: list 36.0pt; mso-list: l0 level1 lfo9"><span lang=EN-US
  style="mso-ansi-language: EN-US">Midori Harris<U1:p></U1:p> </span>
  <li class=MsoNormal
  style="tab-stops: list 36.0pt; mso-list: l0 level1 lfo9"><span lang=EN-US
  style="mso-ansi-language: EN-US">John Quackenbush (TIGR) </span>
  <li class=MsoNormal
  style="tab-stops: list 36.0pt; mso-list: l0 level1 lfo9">JJ<U1:p></U1:p>
  <li class=MsoNormal
  style="tab-stops: list 36.0pt; mso-list: l0 level1 lfo9"><span lang=EN-US
  style="mso-ansi-language: EN-US">Alan Robinson<U1:p></U1:p> </span>
  <li class=MsoNormal
  style="tab-stops: list 36.0pt; mso-list: l0 level1 lfo9"><span lang=EN-US
  style="mso-ansi-language: EN-US">Alphonse Thanaraj </span>
  <li class=MsoNormal
  style="tab-stops: list 36.0pt; mso-list: l0 level1 lfo9"><span lang=EN-US
  style="mso-ansi-language: EN-US">Caleb Webber<b><o:p></o:p></b></span>
</li></ul>
<h3
style="tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt"><span
lang=EN-US style="FONT-WEIGHT: normal; mso-ansi-language: EN-US">For questions
and comments please contact authors<U1:p></U1:p></span></h3>
<h3
style="tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt"><span
lang=EN-US style="mso-ansi-language: EN-US">Further reading
<U1:p></U1:p></span></h3>
<p class=MsoNormal
style="tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt"><span
lang=EN-US style="mso-ansi-language: EN-US"><a
href="http://www.nature.com/cgi-taf/dynapage.taf?file=/nature/journal/v409/n6822/index.html">Nature
&#8211; Human Genome Issue<span style="mso-spacerun: yes">&nbsp; </span>(Volume 409,
p. 813-959, 2001).</a> <br /><a
href="http://www.sciencemag.org/content/vol291/issue5507/">Science &#8211; Human
Genome Issue (Volume 291, 2001</a> ) <br /><a
href="http://www.nature.com/cgi-taf/Dynapage.taf?file=/ng/journal/v21/n1s/index.html">Nature
Genetics &#8211; Chipping Forecast &#8211; devoted to microarrys (Volume 21 supplement 1 -
60, 1999</a> ) <br /><a
href="http://www.elsevier.nl/febs/118/19/show/toc.htt">FEBS Letters &#8211; Functional
Genomics (Volume 480, Nr. 1, August 2000)</a> <U1:p></U1:p></span></p>
<h3
style="tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt"><span
lang=EN-US style="mso-ansi-language: EN-US">Online
resources<U1:p></U1:p></span></h3>
<p class=MsoNormal
style="tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt"><span
lang=EN-US style="mso-ansi-language: EN-US"><a
href="http://www.ncbi.nlm.nih.gov/books/bv.fcgi?call=bv.View..ShowTOC&amp;rid=cell.TOC&amp;depth=2">http://www.ncbi.nlm.nih.gov/books/bv.fcgi?call=bv.View..ShowTOC&amp;rid=cell.TOC&amp;depth=2</a>
<br /><a
href="http://gened.emc.maricopa.edu/bio/bio181/BIOBK/BioBookTOC.html">On-Line
Biology Book</a> <br /><a
href="http://www.biochemweb.org/">The
WWW Virtual Library of Cell Biology</a> <b><o:p></o:p></b></span></p>
<h3><span lang=EN-US style="mso-ansi-language: EN-US">Textbooks on
bioinformatics<U1:p></U1:p></span></h3>
<p class=MsoNormal
style="tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt">Teresa
Attwood, David Parry-Smith, <b>Introduction to Bioinformatics</b> , Prentice
Hall, 1999<span lang=EN-US style="mso-ansi-language: EN-US"><br />Dan Gusfield,
<b>Algorithms on Strings, Trees, and Sequences - Computer Science and
Computational Biology</b>, Cambridge University Press, 1997.<br />Pavel Pevzner,
<b>Computational Molecular Biology &#8211; An Algorithmic Approach</b> , The MIT
press, 2000</span></p>
<h3><span lang=EN-US style="mso-ansi-language: EN-US">Some other popular books
<U1:p></U1:p></span></h3>
<p class=MsoNormal
style="tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt"><span
lang=EN-US style="mso-ansi-language: EN-US">Charles Darwin, <a
href="http://www.literature.org/authors/darwin-charles/the-origin-of-species/"><b>The
Origin of Species</b></a> (theory of evolution)<U1:p></U1:p></span></p>
<p class=MsoNormal
style="tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt"><span
lang=EN-US style="mso-ansi-language: EN-US">Richard Dawkins, <b><a
href="http://www.world-of-dawkins.com/selfish.htm">The Selfish Gene</a><span
style="mso-spacerun: yes">&nbsp;&nbsp;&nbsp; </span></b>(hypotheses of the
&#8216;selfish&#8217; DNA)<U1:p></U1:p></span></p>
<p class=MsoNormal
style="tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt"><span
lang=EN-US style="mso-ansi-language: EN-US">Watson, D. J, <b>The Double
Helix</b>; New York, London, W. W. Norton, 1980 (a personal account of the
discovery of the structure of DNA)<U1:p></U1:p><b><o:p></o:p></b></span></p>
<h3><span lang=EN-US style="mso-ansi-language: EN-US">Popular textbooks on
molecular biology<U1:p></U1:p></span></h3>
<p class=MsoNormal
style="tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt"><span
lang=EN-US style="mso-ansi-language: EN-US">(Molecular biology textbooks are
typically rather big, but usually they mostly consists of nice cartoons through
which one can learn a lot)<U1:p></U1:p></span></p>
<p class=MsoNormal
style="tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt"><span
lang=EN-US style="mso-ansi-language: EN-US">Alberts, B. , Bray, D. , Lewis J. ,
Raff, M., Roberts, K., Watson, J.D.:&nbsp; <b>Molecular biology of the cell.
</b>New York, Garland Publishing, 1994<br />Brown T. A.: <b>Genetics; a molecular
approach</b>, London, Chapman &amp; Hall, 1999&nbsp; <U1:p></U1:p></span></p>
<p class=MsoNormal
style="tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt"><span
lang=EN-US style="mso-ansi-language: EN-US">Lewin B: <b>Genes VII</b>, New York,
Oxford University Press, 2000 <br />Winter, P. C., Hickey, I. , Fletcher, H. L.:
<b>Instant Notes in Genetics</b>, Oxford, 1998 <U1:p></U1:p></span></p>
<h3
style="tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt"><span
lang=EN-US style="mso-ansi-language: EN-US">Other references where some of the
facts on this web-site are taken from </span></h3>
<p class=MsoNormal
style="tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt"><span
lang=EN-US style="mso-ansi-language: EN-US"><a
href="http://nar.oupjournals.org/cgi/content/full/27/15/3219?maxtoshow=&amp;HITS=10&amp;hits=1">Deutsch
and Long, Intron-exon structures of eukaryotic model organism, Nuc Acids Res
(27)15, 3119-3228, 1999</a> <br />Science (294) 81-121, 2001. Unlocking the genome
&#8211; new developments in genomics</span></p>
<p class=MsoNormal
style="tab-stops: 45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt"><b><span
lang=EN-US style="mso-ansi-language: EN-US"><br /></span></b><span lang=EN-US
style="mso-ansi-language: EN-US"><span
style="mso-spacerun: yes">&nbsp;</span><U1:p></U1:p></span><o:p></o:p></p></div><U1:p></U1:p><U1:p></U1:p></body></html>
